src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -270,29 +270,50 @@ extern void setDbPrefix(std::string _dbPrefix)
 *
 * getOrionDatabases -
 */
-extern void getOrionDatabases(std::vector<std::string>& dbs)
+extern bool getOrionDatabases(std::vector<std::string>& dbs)
```

Comment:
Is this function never called anywhere? Because it may need checking whether the return value is true.